cognite/neat/workflows/steps/lib/rules_exporter.py

```diff
@@ -180,3 +180,189 @@ def run(self, rules: MultiRuleData) -> FlowMessage:  # type: ignore[override, sy
         )
 
         return FlowMessage(output_text=output_text)
+
+
+class RulesToOntology(Step):
+    """
+    This step exports Rules to OWL ontology
+    """
+
+    description = "This step exports Rules to OWL ontology"
+    version = "private-beta"
+    category = CATEGORY
+    configurables: ClassVar[list[Configurable]] = [
+        Configurable(
+            name="File path",
+            value="staging/ontology.ttl",
+            label=(
+                "Relative path for the ontology file storage, "
+                "must end with .ttl ! It will be auto-created if not provided !"
+            ),
+        )
+    ]
+
+    def run(self, rules: MultiRuleData) -> FlowMessage:  # type: ignore[override, syntax]
+        if self.configs is None or self.data_store_path is None:
+            raise StepNotInitialized(type(self).__name__)
+
+        if not rules.information and not rules.dms:
+            return FlowMessage(
+                error_text="Rules must be made either by Information Architect or DMS Architect!",
+                step_execution_status=StepExecutionStatus.ABORT_AND_FAIL,
+            )
+        elif rules.dms and not rules.information:
+            information_rules = rules.dms.as_information_architect_rules()
+        else:
+            information_rules = cast(InformationRules, rules.information)
+
+        # ontology file
+        default_path = self.data_store_path / Path(
+            f"{information_rules.metadata.prefix}-"
+            f"v{information_rules.metadata.version.strip().replace('.', '_')}"
+            "-ontology.ttl"
+        )
+
+        if not self.configs["File path"]:
+            storage_path = default_path
+        else:
+            storage_path = self.data_store_path / Path(self.configs["File path"])
+
+        storage_path.parent.mkdir(parents=True, exist_ok=True)
+
+        exporter = exporters.OWLExporter()
+        exporter.export_to_file(storage_path, information_rules)
+
+        relative_ontology_file_path = str(storage_path).split("/data/")[1]
```

Comment:
This does not work on a Windows machine, and it will fail if the configuration `File Path` is `/data/my_file.ttl`

```suggestion
        relative_ontology_file_path = '/'.join(storage_path.relative_to(self.data_store_path).parts)
```

![image](https://github.com/cognitedata/neat/assets/60234212/e0eb7157-a33a-4033-87dc-7a44ecad1859)